### DISCUSSION TITLE
2FA not working

### DISCUSSION THREAD

**Discussion by dridhas** [firefly-iii/firefly-iii#5697] [General]
Hello All,
I've been using Firefly III and loving it.
the only issue that has come up is when enabling or using 2FA.

I'm getting the following error:
Whoops! An error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:
View [v1.auth.mfa] not found.

and the only way to get around it is by going to the DB and running this: **update users set mfa_secret = NULL where email = "your@email.com"** on the DB.

has any of you come across this issue?
I'm running Firefly III v5.6.12 on a Docker container with MariaDB v10.6 also on a Docker container.

Thanks in advance!

**Comment by JC5** [comment]
Upgrade to 5.6.13

**Reply by dridhas** [reply]
Thank you for the update.
I upgraded to 5.6.13 and now I'm able to use 2FA again... :)
Thanks again!
